plugins/outputs/exec/exec.go

```diff
@@ -0,0 +1,113 @@
+package exec
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"os/exec"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers"
+)
+
+var sampleConfig = `
+  ## Command
+  command = "/usr/bin/mycollector --foo=bar"
```

Comment:
probably a nice command taht could be used with the sample config like, "echo"

Should this be an array?